### DISCUSSION TITLE
DEBUG: Authentication | Return-Code Provided is: 404

### DISCUSSION THREAD

**Discussion by 0xRalfR** [elueckel/Unifi-Toolbox#66] [General]
Hello, I once pulled the Unifi Device Monitor onto my SymBox. First of all, thank you for this great module!

The installation went through, the settings too, at least in such a way that no messages came after pressing the buttons:

TEST LOGIN TO THE CONTROLLER
CHECK PRESENCE
CHECK SITE NAME.

Devices created with MAC and name.

In the tree all devices remain "Away" even though they are actively logged into the Unifi network.
In the debug comes the message: Authentication | Return code Provided is: 404

Do the devices have to be in the same network (VLAN)?

Thx

**Comment by Brovning** [comment]
Root cause:
CloudKey 2 used.
CloudKey1 / DreamMachine selected.

Solution:
Settings corrected
--> Issue fixed.
